data/human/free worlds 3 reconciliation.txt

```diff
@@ -1306,7 +1306,7 @@ mission "FW Pug 2C: Jump Drive"
 	to offer
 		has "FW Pug 2B: done"
 		not "fw given jump drive"
-		has "outfit (flagship installed): Jump Drive"
+		has "flagship attribute: jump drive"
```

Comment:
This is just going to lead to someone with a plugin jump drive opening an issue complaining that the game acts like their plugin outfit that is not a "Jump Drive" is called a "Jump Drive" even though it's actually a completely different outfit that does the same thing.